### ISSUE TITLE
Proposal: <podcast:tags> + <podcast:tag>

### ISSUE THREAD

**Issue by coldacid** [Podcastindex-org/podcast-namespace#137]
Alternatively, it could be <podcast:keywords> and <podcast:keyword> but the idea is that we have a more structured format for providing keywords than the <itunes:keywords> comma-delimited format. This would enable scenarios where the keyword/tag itself may require a comma (such as using the title of some other media item that has a comma in it) and should reduce effort on the part of apps, aggregators, search engines, etc. to sort out the different keywords for shows and episodes.

Unlike <itunes:keywords> this should be allowable at both channel and item levels. <itunes:keywords> is apparently only valid at the item level.

Structure would be zero or one <podcast:keywords> elements in the channel, and again zero or one in an item. Within this element would be one or more <podcast:keyword> elements, each element containing an individual keyword/tag value. Empty <podcast:keyword> elements would be invalid. The number of <podcast:keyword> elements within <podcast:keywords> shouldn't be limited by spec, but it should be suggested to keep the number used to no more than 12 and and explicitly allow clients to silently ignore any in excess of that. (Only search functionality would probably find additional keywords/tags beyond the initial 12 to be of any use.)

An example, based on a recent episode of my own show, could be:

```xml
<item>
  <!-- blah blah blah other elements go here -->
  <podcast:keywords>
    <podcast:keyword>moose</podcast:keyword>
    <podcast:keyword>Tonikaku Kawaii</podcast:keyword>
    <podcast:keyword>Digg</podcast:keyword>
    <podcast:keyword>pig couch</podcast:keyword>
  </podcast:keywords>
</item>
```

**Comment by theDanielJLewis** [comment]
I like this idea, but I realized we already have something for this in the RSS spec.
```xml
		<category><![CDATA[Audio]]></category>
		<category><![CDATA[amateur]]></category>
		<category><![CDATA[budget]]></category>
		<category><![CDATA[goals]]></category>
		<category><![CDATA[labels]]></category>
		<category><![CDATA[quality]]></category>
		<category><![CDATA[skill]]></category>
		<category><![CDATA[success]]></category>
```

(I copied that from my own RSS feed, but `CDATA` isn't necessary.)

These `<category>` tags are being added from WordPress's own "tags" feature. So it might actually be unnecessary or redundant to build this into the Podcast namespace.

**Comment by coldacid** [comment]
Is <category> actually part of RSS then or is this something that WordPress just does without a damn for standards?

**Comment by theDanielJLewis** [comment]
`<category>` is in the [RSS spec](https://cyber.harvard.edu/rss/rss.html#ltcategorygtSubelementOfLtitemgt).

> <category>
>
>  sub-element of <item> 
> 
> <category> is an optional sub-element of <item>.
> 
> It has one optional attribute, domain, a string that identifies a categorization taxonomy.
> 
> The value of the element is a forward-slash-separated string that identifies a hierarchic location in the indicated taxonomy. Processors may establish conventions for the interpretation of categories. Two examples are provided below:
> 
> <category>Grateful Dead</category>
> 
> <category domain="http://www.fool.com/cusips">MSFT</category>
> 
> You may include as many category elements as you need to, for different domains, and to have an item cross-referenced in different parts of the same domain.
>

**Comment by coldacid** [comment]
Huh, then that should work.

**Comment by theDanielJLewis** [comment]
But I don't think any podcast app does anything with this. But as a standards group, we can encourage developers to consider support it.

**Comment by coldacid** [comment]
Yeah, we should. At the very least, it's something that should be included in indexing for shows.
